src/main/java/mate/academy/springboot/datajpa/controller/CategoryController.java

```diff
@@ -0,0 +1,45 @@
+package mate.academy.springboot.datajpa.controller;
+
+import lombok.RequiredArgsConstructor;
+import mate.academy.springboot.datajpa.dto.request.CategoryRequestDto;
+import mate.academy.springboot.datajpa.dto.response.CategoryResponseDto;
+import mate.academy.springboot.datajpa.model.Category;
+import mate.academy.springboot.datajpa.service.CategoryService;
+import mate.academy.springboot.datajpa.service.mapper.CategoryMapper;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/categories")
+public class CategoryController {
+    private final CategoryService categoryService;
+    private final CategoryMapper categoryMapper;
+
+    @PostMapping
+    public CategoryResponseDto createCategory(@RequestBody CategoryRequestDto requestDto) {

```

Comment:
```suggestion
    public CategoryResponseDto create(@RequestBody CategoryRequestDto requestDto) {

```
if it is not a specific controller better use simple names.